src/core_plugins/kibana/public/context/api/utils/sorting.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+
+
+function reverseQuerySort(query) {
+  return {
+    ...query,
+    sort: _.get(query, 'sort', []).map(reverseSortDirective),
+  };
+}
+
+function reverseSortDirective(sortDirective) {
+  if (_.isString(sortDirective)) {
+    return {
+      [sortDirective]: (sortDirective === '_score' ? 'asc' : 'desc'),
```

Comment:
I'm aware of the defaults, but other devs might not be. I think it's worth mentioning the reason for this logic in a comment. Particularly because the defaults might change and then this line would be really confusing.